src/DataTable/index.jsx

```diff
@@ -134,6 +148,7 @@ function DataTable({
     disableElevation,
     isLoading,
     isSelectable,
+    manualSelectColumn,
```

Comment:
This needed to be exposed in `DataTableContext` for it to be accessible by the `CardView`.